src/main/scala/org/broadinstitute/dsde/workbench/sam/api/UserRoutesV2.scala

```diff
@@ -197,6 +197,7 @@ trait UserRoutesV2 extends SamUserDirectives with SamRequestContextDirectives {
               Set.empty,
               Set(ResourceRoleName("user")),
               Set.empty,
+              Set.empty,
```

Comment:
Listing all user resources doesn't need to filter on parent, so an empty set is provided here.